dataGenerator/src/test/java/com/filip/datagenerator/DataGeneratorApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.filip.datagenerator;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class DataGeneratorApplicationTests {
+
+    @Test
+    void contextLoads() {
+    }
+
+}
```

Comment:
Delete test module if you don't have tests

Reply:
done